dataverse/webapi/C#-NETx/WebAPIService/WebAPIService.csproj

```diff
@@ -16,4 +16,18 @@
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
   </ItemGroup>
 
+	<!--TODO: You shouldn't need to add these here.
```

Comment:
You shouldn't need to explicitly mention these files for this to work.

Reply:
Ok will remove